airflow/cli/commands/task_command.py

```diff
@@ -437,6 +437,9 @@ def task_test(args, dag=None):
     if args.task_params:
         passed_in_params = json.loads(args.task_params)
         task.params.update(passed_in_params)
+
+    task.params.validate()
```

Comment:
Should we validate always or can we do as below:

```suggestion
    if task.params:
        task.params.validate()
```